combat-trainer.lic

```diff
@@ -352,6 +355,12 @@ class LootProcess
     end
   end
 
+  def should_perform_ritual(game_state)
+    unless game_state.prepare_consume
```

Comment:
Otherwise I think this will try to do a training ritual on the target for the healing spell